streams/src/main/java/org/apache/kafka/streams/kstream/internals/KTablePassThrough.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.kafka.streams.kstream.internals;
+
+import org.apache.kafka.streams.processor.AbstractProcessor;
+import org.apache.kafka.streams.processor.Processor;
+import org.apache.kafka.streams.processor.ProcessorContext;
+import org.apache.kafka.streams.state.TimestampedKeyValueStore;
+import org.apache.kafka.streams.state.ValueAndTimestamp;
+
+import java.util.Collection;
+
+public class KTablePassThrough<K, V> implements KTableProcessorSupplier<K, V, V> {
+    private final Collection<KStreamAggProcessorSupplier> parents;
+    private final String storeName;
+
+
+    KTablePassThrough(final Collection<KStreamAggProcessorSupplier> parents, final String storeName) {
+        this.parents = parents;
+        this.storeName = storeName;
+    }
+
+    @Override
+    public Processor<K, Change<V>> get() {
+        return new KTablePassThroughProcessor();
+    }
+
+    @Override
+    public boolean enableSendingOldValues(final boolean forceMaterialization) {
```

Comment:
It looks like the parameter is ignored. Is that correct? If so, it might be nice to add an explanatory comment.